src/pages/LinkLandingPage/parseQuery.ts

```diff
@@ -19,14 +19,15 @@ function firstOrValue(value: string | string[]) {
 }
 
 export default function parseQuery(query: string): QueryParams {
-  const normalizedQuery = URI.decodeQuery(query).replace("web+comit:swap", "");
+  const queryString = query.replace("?", "");
```

Comment:
:x: I would like this to be more explicit that we only care about the first character. Something like (pseudocode)

if query.startsWith("?")
   query.popFront(1)

Reply:
Please recheck